data-sources/gridgain/config.yml

```diff
@@ -0,0 +1,21 @@
+id: gridgain-data-source
+displayName: GridGain
+description: |
+  Collect the metrics and logs from your GridGain or Apache Ignite cluster using New Relic.
+icon: gridgain.svg
+install:
+  primary:
+    link:
+      url: https://www.gridgain.com/docs/tutorials/new-relic/new-relic-tutorial
+keywords:
+  - database
+  - java
+  - cluster
+  - sql
+  - cache
+  - compute-grid
+  - apache-ignite
+  - ignite
+categoryTerms:
+  - metrics
```

Comment:
Hey @sdarlington, I'm afraid we don't have a `metrics` category term. Perhaps `infrastructure` would fit this data source?